platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -266,12 +282,15 @@ sep10:
   #
   client_attribution_required: false
   ## @param: client_attribution_allow_list
-  ## Set the white list of the client domain. The domains are comma-separated.
-  #
+  ## Set the white list of the client domain. The domains are comma-separated. If a domain is defined in the
+  #  `client_attribution` section of the configuration, the public_key defined in the client_attribution configuration
```

Comment:
I think we should use domain instead of public key, as it's more user friendly

Reply:
Can you help to make the suggestion to elaborate?

Reply:
Sorry please check this comment instead https://github.com/stellar/java-stellar-anchor-sdk/pull/946/files/8be50e02f4c33999fb895b84251f888a043bb32f#r1231540626